js/actions/tests/newSource.js

```diff
@@ -1,25 +1,28 @@
 "use strict";
 
-const { createStore, actions, queries } = require("../../util/test-head");
+const { createStore } = require("../../util/test-head");
+const { newSource } = require("../../actions");
+const { getSourceByActor } = require("../../queries");
+
 const store = createStore();
 var expect = require("chai").expect;
```

Comment:
Do you feel strongly about chai? I prefer https://www.npmjs.com/package/expect, but I don't want to slow this down.